src/Ferrite.jl

```diff
@@ -1,14 +1,27 @@
 module Ferrite
-using Reexport
+
+using Reexport: @reexport
 @reexport using Tensors
 @reexport using WriteVTK
 
+using Base:
+    @propagate_inbounds
+using EnumX:
+    EnumX, @enumx
 using LinearAlgebra
```

Comment:
Since I cannot find it in the Julia docs let me quickly ask here. Is the ":" optional?

Reply:
No, just mistake:
```suggestion
using LinearAlgebra:
```